recommendations/sandbox/keda.md

```diff
@@ -0,0 +1,203 @@
+# CNCF SIG Sandbox Project Request - KEDA
+
+## Acceptance to CNCF Sandbox
+
+**Authors:** Jeff Hollan (@jeffhollan), Tom Kerkhove (@TomKerkhove), Anirudh Garg (@anirudhgarg)
+
+## Background
+
+**_Link to TOC PR_**
+
+https://github.com/cncf/toc/issues/335
+
+**_Link to Presentation_**
+
+Recording: Feb 20 SIG-Runtime meeting
```

Comment:
Can we update the link here as soon as the presentation gets posted on the CNCF channel: https://www.youtube.com/channel/UCvqbFHwN-nwalWPjPUKpvTA? 

cc: @amye 😄

Reply:
We will! Feel free to let us know when you're done!

Reply:
Any update on this @amye?

Reply:
Is this a blocker for merging @raravena80 or can we proceed and add it after the fact?